src/api/Metadata.js

```diff
@@ -1053,6 +1055,80 @@ class Metadata extends File {
 
         return getProp(suggestionsResponse, 'data.suggestions', []);
     }
+
+    /**
+     * Build URL for metadata options associated to a taxonomy field.
+     *
+     * @param scope
+     * @param templateKey
+     * @param fieldKey
+     * @returns {`${string}/metadata_templates/${string}/${string}/fields/${string}/options`}
+     */
+    getMetadataOptionsUrl(scope: string, templateKey: string, fieldKey: string): string {
+        return `${this.getBaseApiUrl()}/metadata_templates/${scope}/${templateKey}/fields/${fieldKey}/options`;
+    }
+
+    /**
+     * Gets options associated with a taxonomy field.
+     *
+     * @param id
+     * @param scope
+     * @param templateKey
+     * @param fieldKey
+     * @param level
+     * @param options
+     * @returns {Promise<MetadataOptions>}
+     */
+    async getMetadataOptions(
+        id: string,
+        scope: string,
+        templateKey: string,
+        fieldKey: string,
+        level: number,
+        options: { marker?: string, searchInput?: string, signal?: AbortSignal },
+    ) {
+        this.errorCode = ERROR_CODE_FETCH_METADATA_OPTIONS;
+
+        if (!id) {
+            throw getBadItemError();
+        }
+
+        if (!scope) {
+            throw new Error('Missing scope');
+        }
+
+        if (!templateKey) {
+            throw new Error('Missing templateKey');
+        }
+
+        if (!fieldKey) {
+            throw new Error('Missing fieldKey');
+        }
+
+        // 0 is a valid level value
+        if (!level && level !== 0) {
+            throw new Error('Missing level');
+        }
+
+        const url = this.getMetadataOptionsUrl(scope, templateKey, fieldKey);
+        const { marker, searchInput, signal } = options;
+        const params = {
+            ...(marker ? { marker } : {}),
+            ...(searchInput ? { searchInput } : {}),
```

Comment:
should `level` be passed in here?

Reply:
Good call, I'll add it in. Hard to test what's required as we only have the mocked response at this time